pkg/ovsdb/monitor.go

```diff
@@ -10,8 +10,10 @@ import (
 	"time"
 
 	"github.com/go-logr/logr"
+	"github.com/pkg/errors"
```

Comment:
probably this is a wrong `errors` package

Reply:
I removed the package,thanks.